packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -37,6 +39,8 @@ contract SortedOracles is ISortedOracles, ICeloVersionedContract, Ownable, Initi
   uint256 public reportExpirySeconds;
   mapping(address => uint256) public tokenReportExpirySeconds;
 
+  IBreakerBox public breakerBox;
```

Comment:
Should the BreakerBox be just registered in the Registry? Then we don't need this variable, don't need `setBreakerBox`, just look up in Registry. Look up from there both here and in Exchange, that way if BreakerBox address changes, it automatically updates for all Exchanges and SortedOracles, no need to remember to manually update it via governance proposal for each contract individually.

Reply:
I initially didn't add it as I didn't want to introduce that dependency. @bowd also suggested it would be best not to add to the registry.

Reply:
This is how we handle inter-core contract dependencies all throughout the protocols. If we break this pattern here, we'll have to remember to manually update this, should the address ever change. This would complicate the release process and create more surface for human error, that the Registry pattern is specifically supposed to help us avoid.

Additionally, registering in the Registry is helpful for some other things, e.g. getting core contract addresses in contractkit.